src/buza/admin.py

```diff
@@ -1,8 +1,26 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import gettext_lazy as _
 
 from buza import models
 
 
 @admin.register(models.User)
-class UserAdmin(admin.ModelAdmin):
-    list_display = ['username', 'school', 'grade']
+class UserAdmin(DjangoUserAdmin):
+    """
+    Extend the base Django UserAdmin with support for some Buza fields.
+    """
+
+    date_hierarchy = 'date_joined'
+    ordering = ['-date_joined']
+    list_display = list(DjangoUserAdmin.list_display) + ['grade', 'date_joined']
+    list_filter = list(DjangoUserAdmin.list_filter) + ['grade']
+
+    fieldsets = list(DjangoUserAdmin.fieldsets[:1]) + [
```

Comment:
what does the `[:1]` mean

Reply:
It's Python's slice notation: `[:1]` means everything up to index 1, and `[1:]` means index 1 and everything after it.

In this case, the code is basically creating a copy of the parent `fieldsets` sequence, but with the additional data (`[…]`) inserted at index 1, like:

```
fieldsets[:1] + […] + fieldsets[1:]
```

(where the additional data is the fieldset definition for the Buza fields)